test/specs/controller.bar.tests.js

```diff
@@ -1372,13 +1372,23 @@ describe('Chart.controllers.bar', function() {
 
 		var meta = chart.getDatasetMeta(1);
 		var bar = meta.data[0];
+		var helpers = window.Chart.helpers;
 
 		// Change default
 		chart.options.elements.rectangle.backgroundColor = 'rgb(128, 128, 128)';
 		chart.options.elements.rectangle.borderColor = 'rgb(15, 15, 15)';
 		chart.options.elements.rectangle.borderWidth = 3.14;
 
-		// Remove to defaults
+		chart.update();
+		expect(bar._model.backgroundColor).toBe('rgb(128, 128, 128)');
+		expect(bar._model.borderColor).toBe('rgb(15, 15, 15)');
+		expect(bar._model.borderWidth).toBe(3.14);
+		meta.controller.setHoverStyle(bar);
+		expect(bar._model.backgroundColor).not.toBe('rgb(128, 128, 128)');
+		expect(bar._model.borderColor).not.toBe('rgb(15, 15, 15)');
+		expect(bar._model.backgroundColor).toBe(helpers.getHoverColor('rgb(128, 128, 128)'));
```

Comment:
Not sure which one is better : 
* `.not.toBe('rgb(128, 128, 128)')`
* `.toBe(helpers.getHoverColor('rgb(128, 128, 128)')`

Reply:
I would say `toBe` is better